src/python_testing/TC_AVSM_2_7.py

```diff
@@ -0,0 +1,339 @@
+                "Store this value in aMinViewport.",
+            ),
+            TestStep(
+                6,
+                "TH reads VideoSensorParams attribute from CameraAVStreamManagement Cluster on TH_SERVER.",
+                "Store this value in aVideoSensorParams.",
+            ),
+            TestStep(
+                7,
+                "TH reads MaxEncodedPixelRate attribute from CameraAVStreamManagement Cluster on TH_SERVER.",
+                "Store this value in aMaxEncodedPixelRate.",
+            ),
+            TestStep(
+                8,
+                "TH sets StreamUsage from aRankedStreamPriorities. TH sets VideoCodec, MinResolution, MaxResolution, MinBitRate, MaxBitRate conforming with aRateDistortionTradeOffPoints. TH sets MinFrameRate, MaxFrameRate conforming with aVideoSensorParams. TH sets the MinFragmentLen and MaxFragmentLen = 4000. TH sends the VideoStreamAllocate command with these arguments.",
+                "DUT responds with VideoStreamAllocateResponse command with a valid VideoStreamID.",
+            ),
+            TestStep(
+                9,
+                "TH reads AllocatedVideoStreams attribute from CameraAVStreamManagement Cluster on TH_SERVER",
+                "Verify the number of allocated video streams in the list is 1.",
+            ),
+            TestStep(
+                10,
+                "TH sends the VideoStreamAllocate command with the same arguments from step 7 except StreamUsage set to a value not in aRankedStreamPriorities.",
+                "DUT responds with a CONSTRAINT_ERROR status code.",
+            ),
+            TestStep(
+                11,
+                "TH sends the VideoStreamAllocate command with the same arguments from step 7 except MinFrameRate set to 0(outside of valid range).",
+                "DUT responds with a CONSTRAINT_ERROR status code.",
+            ),
+            TestStep(
+                12,
+                "TH sends the VideoStreamAllocate command with the same arguments from step 7 except MinFrameRate > MaxFrameRate.",
+                "DUT responds with a CONSTRAINT_ERROR status code.",
+            ),
+            TestStep(
+                13,
+                "TH sends the VideoStreamAllocate command with the same arguments from step 7 except MinBitRate set to 0(outside of valid range).",
+                "DUT responds with a CONSTRAINT_ERROR status code.",
+            ),
+            TestStep(
+                14,
+                "TH sends the VideoStreamAllocate command with the same arguments from step 7 except MinBitRate > MaxBitRate.",
+                "DUT responds with a CONSTRAINT_ERROR status code.",
+            ),
+            TestStep(
+                15,
+                "TH sends the VideoStreamAllocate command with the same arguments from step 7 except MinFragmentLen > MaxFragmentLen.",
+                "DUT responds with a CONSTRAINT_ERROR status code.",
+            ),
+        ]
+
+    @async_test_body
```

Comment:
```suggestion
    @run_if_endpoint_matches(has_feature(Clusters.CameraAvStreamManagement,                                         Clusters.CameraAvStreamManagement.Bitmaps.Feature.kVideo))
```

Reply:
Thank you for your comments. We fixed it with our new commit.